src/webapp/project_reader.py

```diff
@@ -62,8 +60,9 @@ def get_one_project(file: str, campus_grid_ids: Dict, vos_data: VOsData) -> Dict
             ID = vos_data.vos[name]['ID']
             data['Sponsor']['VirtualOrganization'] = OrderedDict([("ID", ID), ("Name", name)])
 
-        if 'ResourceAllocation' in data:
-            data['ResourceAllocation'] = get_resource_allocation(data)
+        if 'ResourceAllocations' in data:
+            resource_allocations = [get_resource_allocation(ra, idx) for idx, ra in enumerate(data['ResourceAllocations'])]
+            data['ResourceAllocations'] = {"ResourceAllocation": resource_allocations}
```

Comment:
As I mentioned on Slack, this data structure confuses me (same thing with the resource groups above). Is the extra dict layer (`ResourceAllocation`/`ResourceGroup`) for the XML generation?

Reply:
Yep it's for the XML. The library that generates XML takes a dict element with a list value and turns it into a series of tags where the name of the tag is the dict key, so the Python data structure that looks like
```
{
 "ul": {
   "li": ["1", "2", "3"]
 }
}
```
turns into
```
<ul>
 <li>1</li>
 <li>2</li>
 <li>3</li>
</ul>
```